x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/document_creation_button.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { EuiButton } from '@elastic/eui';
+import { useActions } from 'kea';
+
+import { DocumentsLogic } from './documents_logic';
+
+export const DocumentCreationButton: React.FC = () => {
+  const { openDocumentCreation } = useActions(DocumentsLogic);
+
+  return (
+    <EuiButton
+      fill={true}
+      color="primary"
+      onClick={openDocumentCreation}
```

Comment:
If you want, feel free to leave this onClick off for now - I can add it as part of my DocumentCreation work

Reply:
Will you be modifying the `openDocumentCreation` action to give it a new signature? If not, I'll just leave it wired up. If you are, then I'll remove this.

Reply:
I'll be removing the `openDocumentCreation` action totally from `DocumentsLogic`, since it will now belong to a new `DocumentsCreationLogic`. Totally up to you if you'd rather remove it now or have me remove it later!

Reply:
In that case I'll just remove it.

Reply:
d88d92ddd6764e5f5497e8c0ddfde6a1d9f9a4a6